src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -57,6 +59,20 @@ public void start() {
         gamerules.get(GameRules.DO_IMMEDIATE_RESPAWN).set(true, server);
         gamerules.get(GameRules.DO_ENTITY_DROPS).set(false, server);
 
+        final var worldBorder = server.getOverworld().getWorldBorder();
+        worldBorder.setSize(Molehunt.CONFIG.getInitialWorldSize());
+        if (Molehunt.CONFIG.getBorderShrinkingStartingTimeOffset() < Molehunt.CONFIG.getGameDuration()) {
+            borderTimeOffsetTime.schedule(new TimerTask() {
+                @Override
+                public void run() {
+                    worldBorder.interpolateSize(
+                            Molehunt.CONFIG.getInitialWorldSize(),
+                            Molehunt.CONFIG.getFinalWorldSize(),
+                            (long) (Molehunt.CONFIG.getGameDuration() - Molehunt.CONFIG.getBorderShrinkingStartingTimeOffset()) * 60 * 60);
+                }
+            }, (long) Molehunt.CONFIG.getBorderShrinkingStartingTimeOffset() * 60 * 60);

```

Comment:
Same comment. But here it must be in millisecond

Reply:
minutes * 60 * 60 = milliseconds so it's fine here

Reply:
No, one second is 1000 milliseconds